### PR TITLE
Allow users to input guided answer feedback

### PR DESCRIPTION
**Pull request by sanat-thukral**
#  Allow users to input whether the guided answer 'solved' or 'not solved' their issue 

#96 

## Description

Added a way for users to provide feedback if the guide solved their issue. If not then, we provide them other options. There is no data returned to the API at the moment.

## Checklist for Pull Requests

- [x] Supplied as many details as possible on this change
- [x] Included the link to the associated issue in the Issue section above
- [x] The code conforms to the general [development guidelines](https://github.com/SAP/guided-answers-extension/blob/main/docs/developer-guide.md).
- [ ] The code has **unit tests** where applicable and is easily unit-testable
- [x] This branch is appropriately named for the associated issue

### CODE REVIEW

File: packages/webapp/src/webview/ui/components/GuidedAnswerNode/GuidedAnswerNode.tsx [hunk 1]
```diff
@@ -103,19 +129,35 @@ function getContent(activeNode: GuidedAnswerNodeType): ReactElement {
...
+                )}
             </div>
         </div>
     );
+
+    const middle = appState.guideFeedback === false ? middleNotSolved : middleStandard;
+
+    // );
+
+    // const notSolvedButtons = [
```
Comment (unseen1980) [inline packages/webapp/src/webview/ui/components/GuidedAnswerNode/GuidedAnswerNode.tsx:154]: Remove?

File: packages/webapp/src/webview/ui/components/GuidedAnswerNode/GuidedAnswerNode.tsx [hunk 2]
```diff
@@ -92,7 +93,32 @@ function getNavigationSection(): ReactElement {
...
+        { link: 'https://launchpad.support.sap.com/#/expertchat/create', text: 'Open an Incident' },
+        { link: 'https://launchpad.support.sap.com/#/expertchat/create', text: 'Ask the SAP Community' }
+    ];
+
+    const middleNotSolved = (
+        <div id="middle" className="column">
+            <h1>Issue is not resolved</h1>
+            <div id="hr"></div>
+            <p className="guided-answer__node__question">
+                <strong>We are sorry to hear that your issue is not yet resolved.</strong>
```
Comment (unseen1980) [inline packages/webapp/src/webview/ui/components/GuidedAnswerNode/GuidedAnswerNode.tsx:109]: i18n here?

File: packages/webapp/src/webview/ui/components/FeedbackInterface/FeedbackInterface.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import Solved from './images/solved.svg';
+import i18next from 'i18next';
+import { actions } from '../../../state';
+import './FeedbackInterface.scss';
+import { SolvedMessage } from '../SolvedMessage';
+
+export function FeedbackInterface(): ReactElement {
+    return (
+        <div className="feedback-container">
+            <h3>Please tell us if this answer was helpful</h3>
```
Comment (unseen1980) [inline packages/webapp/src/webview/ui/components/FeedbackInterface/FeedbackInterface.tsx:12]: i18n here?

File: packages/webapp/src/webview/ui/components/FeedbackInterface/FeedbackInterface.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        color: var(--vscode-settings-headerForeground);
+        font-size: 14px;
+        font-weight: 700;
+        margin-bottom: 0px;
+    }
+
+    margin-bottom: 15px;
+}
+
+.solved-box {
```
Comment (unseen1980) [inline packages/webapp/src/webview/ui/components/FeedbackInterface/FeedbackInterface.scss:19]: There is css rules duplication in solved-box and not-solved-box classes. Can we have common css rules instead, please?

**Comment by sanat-thukral** [comment]
@unseen1980 I believe all code review changes have been addressed. Thanks for the review :)

**Comment by Klaus-Keller** [comment]
This is continued in https://github.com/SAP/guided-answers-extension/pull/259, closing this PR.

**Pull request closed without merging**